pandaharvester/harvesterbody/job_fetcher.py

```diff
@@ -46,16 +46,19 @@ def run(self):
                     nJobs = harvester_config.jobfetcher.maxJobs
                 # get jobs
                 tmpLog.debug('getting {0} jobs'.format(nJobs))
+                start_getJobs = time.time()
```

Comment:
This can be replaced with

sw = core_utils.get_stopwatch()

and 

sw.get_elapsed_time()

See https://github.com/PanDAWMS/panda-harvester/blob/master/pandaharvester/harvestercore/core_utils.py#L371